### ISSUE TITLE
Yank ver 0.17 causes errors in the hydration phenol example

### ISSUE THREAD

**Issue by nividic** [choderalab/yank-examples#59]
I made a fresh conda installation on Yank and I was trying to run the hydration phenol example in explicit water. Unfortunately I got the following error on OSX and Ubuntu linux: 

Running simulation...
2017-08-23 12:45:39,347: This device does not support double precision for OpenCL. Setting OpenCL precision to 'single'
2017-08-23 12:45:39,347: Setting OpenCL platform to use precision model 'single'.
2017-08-23 12:45:39,816: Single node: executing <function ExperimentBuilder._check_resume at 0x113cfb268>
2017-08-23 12:45:39,817: Single node: executing <function ExperimentBuilder._setup_experiments at 0x113cfb488>
2017-08-23 12:45:39,817: Setting up the systems for phenol, water and vacuum
Traceback (most recent call last):
  File "/Users/gcalabro/local/miniconda3/envs/yank/bin/yank", line 11, in <module>
    load_entry_point('yank==0.17.0', 'console_scripts', 'yank')()
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/cli.py", line 71, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/commands/script.py", line 113, in dispatch
    yaml_builder.run_experiments()
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/experiment.py", line 694, in run_experiments
    self._setup_experiments()
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/mpi.py", line 250, in _wrapper
    return run_single_node(rank, task, *args, **kwargs)
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/mpi.py", line 199, in run_single_node
    result = task(*args, **kwargs)
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/experiment.py", line 1753, in _setup_experiments
    self._db.get_system(system_id)
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/pipeline.py", line 1014, in get_system
    0, system_parameters, solvent1_id, solute_id)
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/pipeline.py", line 1455, in _setup_system
    warnings = tleap.run()
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/utils.py", line 1735, in run
    raise RuntimeError(error_msg + ' Check log file {}'.format(log_path))
RuntimeError: Could not create one of the system files. Check log file /Users/gcalabro/Projects/Yank/yank-examples/examples/hydration/phenol/setup/systems/hydration-system/solvent1.leap.log
Analyzing data...
2017-08-23 12:45:42,488: ERROR - yank.analyze - Cannot find analysis.yaml script in experiments
Traceback (most recent call last):
  File "/Users/gcalabro/local/miniconda3/envs/yank/bin/yank", line 11, in <module>
    load_entry_point('yank==0.17.0', 'console_scripts', 'yank')()
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/cli.py", line 71, in main
    dispatched = getattr(commands, command).dispatch(command_args)
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/commands/analyze.py", line 81, in dispatch
    analyze.analyze_directory(args['--store'])
  File "/Users/gcalabro/local/miniconda3/envs/yank/lib/python3.5/site-packages/yank/analyze.py", line 1378, in analyze_directory
    raise RuntimeError(err_msg)
RuntimeError: Cannot find analysis.yaml script in experiments

**Comment by Lnaden** [comment]
@nividic I have not had a chance to finish updating the yank examples repo yet to use the new YANK, so there may be some settings which will cause errors. The actual error is this one:

```
RuntimeError: Could not create one of the system files. Check log file /Users/gcalabro/Projects/Yank/yank-examples/examples/hydration/phenol/setup/systems/hydration-system/solvent1.leap.log
```

Check that and see what it says. My initial guess would be that the water parameters being loaded are tip3p (old examples), but we moved to tip4p-ew as the default water model. Check your leap parameter options and make sure the water model is `leaprc..water.tip4pew`. This is my initial guess at what is wrong, but the log file should tell you more.

**Comment by nividic** [comment]
@Lnaden Yes the problem was the water model specified in the explicit.yaml file. This should be 

leaprc.water.tip4pew

Thanks

**Comment by jchodera** [comment]
We should really get those nightly travis tests running again, but I think that requires us to speed up the minimizer somehow.

**Comment by Lnaden** [comment]
I'm going to mark this as resolved for now, I have opened #60 to try and catch these problems going forward.
